### PR TITLE
Fixed hindi version

### PR DESCRIPTION
**Pull request by Midnight1938**
Added proper translations to the Hindi version of the app.

### CODE REVIEW

**Comment by brahmkshatriya** [comment]
man simply copypasted into google translate, hoping everything would translate well

**Comment by moenkaey** [comment]
man atleast put some effort in making it understandable.

**Comment by brahmkshatriya** [comment]
> > man simply copypasted into google translate, hoping everything would translate well
> 
> Mans studying at the Central Hindi Institute...

that does make alot of sense in context to your pr

File: app/src/main/res/values-hi/strings.xml [hunk 1]
```diff
@@ -1,170 +1,170 @@
...
+    <string name="watch_quality_pref">प्रिय गुणवत्ता</string>
+    <string name="dns_pref_summary">ISP अवरुद्धता से छुटकारा पाएं</string>
+    <string name="provider_lang_settings">सूत्र भाषा</string>
+    <string name="app_layout">आवेद का विन्यास</string>
+    <string name="preferred_media_settings">प्रिय साधन</string>
     
-    <string name="download_path_pref"> डाउनलोड करने का मार्ग</string>
-    <string name="display_subbed_dubbed_settings">Dubbed या Subbed Anime दिखाये</string>
+    <string name="download_path_pref">डाउनलोड का मार्ग</string>
+    <string name="display_subbed_dubbed_settings">अनुवाद या उपशी एनिमे दिखाये</string> !!!!!!!!!!!!!!!!
```
[the remark this answers is not in the document]
Comment (Midnight1938) [reply app/src/main/res/values-hi/strings.xml]: Sorry about that I was using exclamations to mark things

**Comment by Midnight1938** [comment]
> > > man simply copypasted into google translate, hoping everything would translate well
> > 
> > Mans studying at the Central Hindi Institute...
> 
> that does make alot of sense in context to your pr

All i mean is using google translate is useless when you've been speaking it all your life.

**Comment by Blatzar** [comment]
Unfortunately as my Indian associates has deemed this translation substandard I cannot merge this in good faith :pensive:

**Comment by Midnight1938** [comment]
> Unfortunately as my Indian associates has deemed this translation substandard I cannot merge this in good faith pensive

Do understand they are using the hindi version as a novelty and not as a necessity

**Comment by brahmkshatriya** [comment]
this man makes me laugh each time he writes something ☸️☸️☸️☸️☸️

**Comment by OshekharO** [comment]
This translation is much better for people who only knows Hindi. Whoever translated this in Hindi at the beginning have just written the English word in Hindi.

**Comment by Midnight1938** [comment]
> this man makes me laugh each time he writes something

**Pull request closed without merging**